diff --git a/core/control-plane/env.ts b/core/control-plane/env.ts
index 6abc4d3..5fdf4ab 100644
--- a/core/control-plane/env.ts
+++ b/core/control-plane/env.ts
@@ -7,7 +7,7 @@ import {
 import { AuthType, ControlPlaneEnv } from "./AuthTypes";
 import { getLicenseKeyData } from "./mdm/mdm";
 
-export const EXTENSION_NAME = "continue";
+export const EXTENSION_NAME = "zodeve";
 
 const WORKOS_CLIENT_ID_PRODUCTION = "client_01J0FW6XN8N2XJAECF7NE0Y65J";
 const WORKOS_CLIENT_ID_STAGING = "client_01J0FW6XCPMJMQ3CG51RB4HBZQ";
